src/renderer/src/components/settings-modal/settings-modal.tsx

```diff
@@ -241,13 +252,19 @@ const SettingsModal: React.FC<SettingsModalProps> = ({ closeModal }) => {
                     onChange={(e) => debouncedPassword(e.target.value)}
                   ></input>
 
-                  {/* <label className="mt-1">Radio Effects</label>
-                  <select id="" className="form-control mt-1" disabled>
+                  <h5 className="mt-4">Other</h5>
```

Comment:
I removed the header previously and would rather not have another header in the settings dialog, can you remove?